src/plugins/recordTypes/summarydocumentation/fields.js

```diff
@@ -427,7 +427,10 @@ export default (configContext) => {
               }),
               repeating: true,
               view: {
-                type: CompoundInput,
+                type: TermPickerInput,
+                props: {
+                  source: 'documentationgroup',
+                },
```

Comment:
I think this change is on the wrong field. Did you mean to put it on statusGroupType?

Reply:
Yep looks like that was put on the wrong field. Just pushed a commit to fix this.